chromium_src/ios/CPPLINT.cfg

```diff
@@ -0,0 +1,2 @@
+# Common linting false posititives from Obj-C syntax in headers
+filter=-readability/casting,-whitespace/parens,-whitespace/operators
```

Comment:
why do we have an issue with this if upstream doesn't have it in their config?

Reply:
Im not sure, we have the same issue in our own `ios` folder https://github.com/brave/brave-core/blob/master/ios/CPPLINT.cfg

Reply:
what specifically is triggering it?

Reply:
```src/brave/chromium_src/ios/web_view/internal/cwv_web_view_configuration_internal.h:34:  (cpplint) Using C-style cast.  Use reinterpret_cast<CWVWebView*>(...) instead  [readability/casting] [4]
src/brave/chromium_src/ios/web_view/internal/cwv_web_view_internal.h:15:  (cpplint) Using C-style cast.  Use reinterpret_cast<NSData*>(...) instead  [readability/casting] [4]
** Presubmit Messages: 1 **
```

The actual lines it references don't make sense. For instance the first one is just a standard Obj-C method:

```- (void)registerWebView:(CWVWebView*)webView;```

Reply:
cc @goodov if you have any ideas?